config/routes.rb

```diff
@@ -2,7 +2,9 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   get 'application/home'
 
-  resources :images, only: %i[index new create show destroy]
+  resources :images do
+    resources :image_shares, only: %i[new create]
```

Comment:
I think singular resources would do here

Reply:
@lsq921224 , do you mean that `image_shares` (plural) become `image_share` (singular)?

Reply:
I meant `resource` over `resources`, it's not a big deal here though since you specified `only`